array/DNAm/preprocessing/CETYGOdeconvolution.r

```diff
@@ -0,0 +1,195 @@
+##---------------------------------------------------------------------#
+##
+## Title: Run CETYGO cell type deconvolution
+##
+## Purpose of script: Use CETYGO brain ref panal to run CETYGO on the samples
+##
+## Author: Emma Walker
+##
+## Date Created: Feb 2024
+##
+##---------------------------------------------------------------------#
+
+
+#----------------------------------------------------------------------#
+# NOTES
+#----------------------------------------------------------------------#
+# project folder is provided on command line
+# More info on the CETYGO package can be found here: 
+#   https://github.com/ejh243/CETYGO/wiki/Deconvolution-of-brain-cell-types
+
```

Comment:
Add a note that this is doing the deconvolution on the raw data for the purpose of QC?
Do we need to think about setting it up so it can be run on either raw or norm data? (I know we discussed that in reality it was overkill but we wanted to provide the option initially even if we later advise against it)

Reply:
added this as an issue